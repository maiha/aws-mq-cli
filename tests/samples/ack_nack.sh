#!/bin/bash

aws-mq-cli samples ack_nack run

