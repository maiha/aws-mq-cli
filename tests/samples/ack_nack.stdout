 [x] Published messages containing a number or a string.
 [*] Consuming with ack for numbers. (To exit press CTRL+C)
 [x] Received [100] (ack)
 [x] Received [foo] (nack:requeue)
 [x] Received [200] (ack)
 [x] Received [foo] (nack:requeue)
